python/lsst/analysis/tools/actions/plot/focalPlanePlot.py

```diff
@@ -168,8 +183,14 @@ def makePlot(
             noDataFig = addPlotInfo(noDataFig, plotInfo)
             return noDataFig
 
-        fig = plt.figure(dpi=300)
-        ax = fig.add_subplot(111)
+        if self.addHistogram:
+            fig = plt.figure(dpi=300, figsize=(12, 6))
```

Comment:
Any particular reason not to call `plt.subplots(...)` instead?

Reply:
I would generally use `plt.subplots()`, but @mfisherlevine and I pair-coded this, and he has strong negative feelings about using `plt.<anything>` other than `plt.figure()`. After the fact, I think we determined that `plt.subplots()` is effectively no worse than `plt.figure()`, so I may go back to this.

Reply:
Ok, I thought subplots just conveniently made more or less the same calls as what you had there anyway.